datafusion/core/src/execution/context.rs

```diff
@@ -1907,7 +1776,18 @@ impl ContextProvider for SessionState {
     }
 
     fn get_config_option(&self, variable: &str) -> Option<ScalarValue> {
-        self.config_options().get(variable)
+        // TOOD: Move ConfigOptions into common crate
+        match variable {
+            "datafusion.execution.time_zone" => self
```

Comment:
This is a nasty hack, I hope to move `ConfigOptions` into core in a subsequent PR so that it can be used directly instead of via `ContextProvider`